### PR TITLE
Add responsiveness for why contribute section

### PR DESCRIPTION
**Pull request by AdelinaMoroaca**

### CODE REVIEW

File: style/scss/_why-contribute.scss [hunk 1]
```diff
@@ -50,3 +50,153 @@
...
+        width: 18.3rem;
+        height: 2rem;
+      }
+    }
+
+    &__description {
+      padding: 0 4rem;
+      
+      & p {
+        font-size: 1.5rem;
```
Comment (talidag) [inline style/scss/_why-contribute.scss:162]: The font size should be either 16 or 14. Also, for the sizes smaller than 320px, the font size is 16px, while this one is 15px.

File: style/scss/_why-contribute.scss [hunk 2]
```diff
@@ -50,3 +50,153 @@
...
+      gap: 2.4rem;
+      padding-bottom: 0rem;
+
+      &__reason {
+        padding: 0rem;
+        gap: 0.9rem;
+  
+        & p {
+          font-size: 1.4rem;
+          line-height: 135.9%;
```
[the remark this answers is not in the document]
Comment (AdelinaMoroaca) [reply style/scss/_why-contribute.scss:185]: done (changed to 136%)

File: style/scss/_why-contribute.scss (hunk 1 above)
Comment (AdelinaMoroaca) [reply style/scss/_why-contribute.scss:162]: done (changed to 1.6rem)

File: style/scss/_why-contribute.scss [hunk 3]
```diff
@@ -50,3 +50,153 @@
...
+
+      &__reason {
+        padding: 0rem;
+  
+        & img {
+          width: 7rem;
+        }
+
+        & p {
+          font-size: 1.9rem;
```
[the remark this answers is not in the document]
Comment (AdelinaMoroaca) [reply style/scss/_why-contribute.scss:83]: done (changed to 1.8 rem))

File: style/scss/_why-contribute.scss [hunk 4]
```diff
@@ -50,3 +50,153 @@
...
+@media screen and (min-width: $laptop-size) and (max-width: 1398px){
+  .why-contribute {
+    &__title {
+      padding: 3.5rem 0;
+    }
+
+    &__description {
+      padding: 0 15rem 4rem 15rem;
+      
+      & br {
```
[the remark this answers is not in the document]
Comment (AdelinaMoroaca) [reply style/scss/_why-contribute.scss]: removed br and added 2 separate paragraphs

**Review by talidag — APPROVED** [review]

**Pull request merged**